### PR TITLE
fixes f5 modules specific redirects

### LINKED ISSUE #69862: Incorrect entries in ansible_builtin_runtime.yml file
##### SUMMARY

Incorrect entries in `ansible_builtin_runtime.yml`

Problematic lines:
https://github.com/ansible/ansible/blob/devel/lib/ansible/config/ansible_builtin_runtime.yml#L2163 to
https://github.com/ansible/ansible/blob/devel/lib/ansible/config/ansible_builtin_runtime.yml#L2178

The entries should be as below:

```
    bigip_device_facts:
      redirect: f5networks.f5_modules.bigip_device_info
    bigip_iapplx_package:
      redirect: f5networks.f5_modules.bigip_lx_package
    bigip_security_address_list:
      redirect: f5networks.f5_modules.bigip_firewall_address_list
    bigip_security_port_list:
      redirect: f5networks.f5_modules.bigip_firewall_port_list
    bigip_traffic_group:
      redirect: f5networks.f5_modules.bigip_device_traffic_group
    bigip_facts:
      tombstone:
        removal_date: 2019-11-06
        warning_text: bigip_facts has been removed please use bigip_device_info module
    bigip_gtm_facts:
      tombstone:
        removal_date: 2019-11-06
        warning_text: bigip_gtm_facts has been removed please use bigip_device_info module
```
##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

lib/ansible/config/ansible_builtin_runtime.yml
##### ANSIBLE VERSION

```paste below

```

##### CONFIGURATION

```paste below

```

##### STEPS TO REPRODUCE

```yaml

```

##### ACTUAL RESULTS

```paste below

```

### PR DESCRIPTION
**Pull request by wojtek0806**
##### SUMMARY

corrects f5 specific module redirects

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

 lib/ansible/config/ansible_builtin_runtime.yml
##### ADDITIONAL INFORMATION

fixes #69862 

```paste below

```

### CODE REVIEW

File: lib/ansible/config/ansible_builtin_runtime.yml [hunk 1]
```diff
@@ -2160,24 +2160,24 @@ plugin_routing:
       redirect: community.network.exos_lldp_interfaces
     exos_vlans:
       redirect: community.network.exos_vlans
-    bigip_asm_policy:
```
Comment (gundalow) [inline lib/ansible/config/ansible_builtin_runtime.yml:2163]: What's the state of this, does it need tombstoning?

Comment (wojtek0806) [reply lib/ansible/config/ansible_builtin_runtime.yml:2163]: yeah it is tombstoned, fixing this now

**Review by gundalow — APPROVED** [review]

**Comment by mattclay** [comment]
Reverting this in https://github.com/ansible/ansible/pull/69874 since this introduces test failures.

**Comment by mattclay** [comment]
@nitzmahone and I are going to discuss how to handle dates in the runtime metadata today. Hopefully we'll have a solution soon so that these changes can be re-applied.

**Comment by mattclay** [comment]
I've opened a PR to re-add these changes: https://github.com/ansible/ansible/pull/69921

The bug that was preventing this change from passing tests was fixed in: https://github.com/ansible/ansible/pull/69918

**Pull request merged**